src/js/components/ServiceOptions.js

```diff
@@ -63,14 +64,19 @@ class ServiceOptions extends mixin(StoreMixin) {
   }
 
   getServiceOptionButtons(service) {
+    let {props, state} = this;
+
     let editButtonClasses = classNames('button button-inverse button-stroke', {
-      'disabled': !this.state.packageFetched
+      'disabled': !state.packageFetched
     });
 
     let buttons = [
       <a className={editButtonClasses} key="edit-config"
         onClick={this.handleEditConfigClick}>
         Edit
+        <UpdateConfigModal open={state.editConfigModalOpen}
```

Comment:
Did we decide to do this consistently at some point? TBH I'm not a fan of one prop per line, but if that's what we decided then I'll stick to it